src/Context.tsx

```diff
@@ -13,18 +13,23 @@ export interface RestfulReactProviderProps<T = any> {
    * Options passed to the fetch request.
    */
   requestOptions?: (() => Partial<RequestInit>) | Partial<RequestInit>;
+  /**
+   * The initial base path given to the parent.
+   */
+  originalBase?: string;
 }
 
 const { Provider, Consumer: RestfulReactConsumer } = React.createContext<RestfulReactProviderProps>({
   base: "",
+  originalBase: undefined,
```

Comment:
I made this `undefined` to make the existence check syntactically simpler, no other reason than that.

Reply:
This is a comment you'd leave in code since you're the author, as opposed to on GitHub where it goes away with the PR. 🙂